plugins/tag/commands

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$PLUGIN_PATH/common/functions"
+source "$(dirname $0)/functions"
+
+case "$1" in
+  tag:add)
```

Comment:
Maybe `tag:create` would be a better name (same as `apps:create` and all the other services).